src/patch.rs

```diff
@@ -106,46 +106,53 @@ pub struct Patch {
     /// The target is `[patch.TARGET]` in the final `Cargo.toml`.
     #[structopt(
         long,
-        conflicts_with_all = &[ "crates", "substrate", "cumulus", "polkadot", "beefy" ]
+        conflicts_with_all = &[ "crates", "substrate", "cumulus", "polkadot", "beefy", "sdk" ]
     )]
     target: Option<String>,
 
+    /// Use the official Polkadot-SDK monorepo as patch target.
+    #[structopt(
+    long,
+    conflicts_with_all = &[ "target", "substrate", "polkadot", "cumulus", "beefy", "crates" ]
+)]
+    sdk: bool,
```

Comment:
We don't need `substrate`, `polkadot` etc anymore as everything is now in the mono repo. Can you remove them?

Reply:
sure, will do asap

Reply:
well, ditching support for all old repositories would only leave one possible workflow, therefore I will remove the need for any option and just support `polkadot-sdk`

Reply:
There is still the `crates` option?

Reply:
yes, I just pushed the changes